File: app/src/main/java/com/exam/main/CPagerAdapter.java
package com.exam.main;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;
import android.view.View;

import java.util.List;

/**
 * @author LowAndroider
 * @date 16:13
 */

public class CPagerAdapter extends FragmentPagerAdapter {

    private List<Fragment> fragments;

    public CPagerAdapter(FragmentManager fm, List<Fragment> fragments) {
        super(fm);
        this.fragments = fragments;
    }


    @Override
    public int getCount() {
        return fragments.size();
    }

    @Override
    public Fragment getItem(int i) {
        return fragments.get(i);
    }
}

File: app/src/main/java/com/exam/main/CPagerAdapter.kt
package com.exam.main

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentPagerAdapter
import android.view.View

/**
 * @author LowAndroider
 * @date 16:13
 */

class CPagerAdapter(fm: FragmentManager, private val fragments: List<Fragment>) : FragmentPagerAdapter(fm) {


    override fun getCount(): Int {
        return fragments.size
    }

    override fun getItem(i: Int): Fragment {
        return fragments[i]
    }
}
